### PR TITLE
Crn 1279 enable pm's and asap staff to edit wg shared research

### PR DESCRIPTION
**Pull request by oratiu**

### CODE REVIEW

**Review by kmaid — APPROVED** [review]

File: packages/validation/src/permissions/research-output.ts [hunk 1]
```diff
@@ -31,3 +35,62 @@ export const hasWorkingGroupsCreateUpdateResearchOutputPermissions = (
...
+
+export const noPermissions: userPermissions = {
+  createDraft: false,
+  editDraft: false,
+  publishDraft: false,
+  editPublished: false,
+};
+
+export const isUserAsapStaff = (user: Omit<User, 'algoliaApiKey'>): boolean =>
+  user.teams.some((team) => team.role === 'ASAP Staff');
```
Comment (gabiayako) [inline packages/validation/src/permissions/research-output.ts:47]: I believe this is not how we want to verify if the user is staff, this function is now verifying the role inside a team

Screenshot 2023-02-21 at 13 57 51

but from the ticket
Screenshot 2023-02-21 at 13 59 11

it seems that we actually want to check this another field `ASAP Hub Role` from `users` entity
Screenshot 2023-02-21 at 13 57 04

File: apps/crn-frontend/src/shared-research/state.ts [hunk 2]
```diff
@@ -160,3 +163,11 @@ export const useSetResearchOutputItem = () => {
     setRefresh(refresh + 1);
   };
 };
+
+export const useUserPerrmisions = (
```
Comment (gabiayako) [inline apps/crn-frontend/src/shared-research/state.ts:167]:
```suggestion
export const useUserPermissions = (
```

**Review by gabiayako — APPROVED** [review]

File: apps/crn-frontend/src/shared-research/state.ts (hunk 2 above)
Comment (oratiu) [reply apps/crn-frontend/src/shared-research/state.ts:167]: that typo tho, I have to find the spelling vsCode plugin Tom told me about

**Pull request merged**